visualization/.gitignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
This is meant to be checked in for reproducible builds. If you want to ignore it, delete the existing instances

Reply:
I meant to delete this file